code/modules/reagents/reagents/reagents_toxin.dm

```diff
@@ -479,6 +479,51 @@
 		O.visible_message("<span class='warning'>\The [O] melts.</span>")
 		qdel(O)
 
+/datum/reagent/mutagen/metastable
+	name = "Metastable Mutagen"
+	id = METASTABLE_MUTAGEN
+	description = "Causes controlled mutations in plants and tables."
+
+/datum/reagent/mutagen/metastable/New()
+	..()
+	name = pick("Metatable Mutagen", "Metastable Mutagen") //it's funny to me
```

Comment:
Why do this

Reply:
Funny to me (it's right there in the comment)